Do not let the btls build pick up ccache from PATH behind MacPorts' back.
https://trac.macports.org/wiki/UsingTheRightCompiler
--- a/mono/btls/CMakeLists.txt
+++ b/mono/btls/CMakeLists.txt
@@ -12,10 +12,6 @@
 enable_language(C)
 enable_language(CXX)
 
-find_program(CCACHE_PROGRAM ccache)
-if(CCACHE_PROGRAM)
-    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE "${CCACHE_PROGRAM}")
-endif()
 
 if (MSVC OR CYGWIN)
 	set(BTLS_HOST_WIN32 1)
